xml/System.Linq/Enumerable.xml

```diff
@@ -466,17 +468,19 @@
         </Parameter>
       </Parameters>
       <Docs>
-        <typeparam name="TSource">To be added.</typeparam>
-        <typeparam name="TKey">To be added.</typeparam>
-        <typeparam name="TAccumulate">To be added.</typeparam>
-        <param name="source">To be added.</param>
-        <param name="keySelector">To be added.</param>
-        <param name="seed">To be added.</param>
-        <param name="func">To be added.</param>
-        <param name="keyComparer">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="TSource">The type of the elements of <paramref name="source" />.</typeparam>
+        <typeparam name="TKey">The type of the key returned by <paramref name="keySelector" />.</typeparam>
+        <typeparam name="TAccumulate">The type of the accumulator value.</typeparam>
+        <param name="source">An <see cref="T:System.Collections.Generic.IEnumerable`1" /> to aggregate over.</param>
+        <param name="keySelector">A function to extract the key for each element.</param>
+        <param name="seed">The initial accumulator value.</param>
+        <param name="func">An accumulator function to be invoked on each element.</param>
+        <param name="keyComparer">An <see cref="T:System.Collections.Generic.IEqualityComparer`1" /> to compare keys with.</param>
+        <summary>Applies an accumulator function over a sequence, grouping results by key.</summary>
+        <returns>An enumerable containing the aggregates corresponding to each key deriving from <paramref name="source" />.</returns>
+        <remarks>
+          <para>This method is comparable to the <see cref="M:System.Linq.Enumerable.GroupBy``2(System.Collections.Generic.IEnumerable{``0},System.Func{``0,``1})" /> methods where each grouping is being aggregated into a single value as opposed to allocating a collection for each group.</para>
+        </remarks>
```

Comment:
Can you update your tool version to the latest? These paras for individual sentences were a bug, and I fixed them in the latest version.

```suggestion
        <remarks>This method is comparable to the <see cref="M:System.Linq.Enumerable.GroupBy``2(System.Collections.Generic.IEnumerable{``0},System.Func{``0,``1})" /> methods where each grouping is being aggregated into a single value as opposed to allocating a collection for each group.</remarks>
```

Reply:
I installed the tool today using the latest commit (https://github.com/dotnet/api-docs-sync/commit/467397badf50362f9086f7d6d218791b80fb9ec1)

Reply:
Okay, I'll investigate.